features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/WireframeMapper.kt

```diff
@@ -22,9 +24,16 @@ interface WireframeMapper<in T : View, out S : MobileSegment.Wireframe> {
      * Maps a [View] to a [List<Wireframe>] in order to be rendered in the Session Replay player.
      * @param view as the [View] instance that will be mapped
      * @param mappingContext in which we provide useful information regarding the current
-     * system state.
+     * @param asyncJobStatusCallback a callback that can be called when the mapper starts or
+     * finishes processing an async job. This is useful to keep track of the current async jobs
```

Comment:
since both `jobStarted` and `jobFinished` doesn't have any arguments like job ID and result, maybe we can elaborate why it is useful? Without any arguments passed it can probably be served as a counter only.